templates/highlighter.css

```diff
@@ -0,0 +1,21 @@
+.__popcode-highlighter {
+  z-index: 2000000;
```

Comment:
I’m not an enormous fan of arbitrarily large `z-indexes` to accomplish “definitely above everything else”—when we’re constructing the highlighter, can we just find the topmost ancestor element that has a `z-index`, and set the highighter’s `z-index` to one plus that?